README.md

```diff
@@ -26,6 +26,8 @@ the raw metrics. Note that the metrics exposed on the `/metrics` endpoint
 reflect the current state of the Kubernetes cluster. When Kubernetes objects
 are deleted they are no longer visible on the `/metrics` endpoint.
 
+Note that any new features will be merged into master but released with v2.1.0 release, as currently v2.0.0 is in post feature freeze and only accepting bug fixes.
```

Comment:
I think this is too specific to the currrent timeline and will become obsolete once we ship v2.1.0. I feel we're better off excluding this from the doc

Reply:
The problem is, people like to know when their feature lands and I would like to point to something or better yet avoid having to answer the questions as its answered in the README. Can you think of a better way?

Reply:
with the amount of questions that we've had around this, I'm also for a statement like this being here, I think it will help with the support burden around this